.circleci/build-debian.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+#
+# Build for Debian in a docker container
+#
+
+# bailout on errors and echo commands.
+set -xe
+
+DOCKER_SOCK="unix:///var/run/docker.sock"
+
+echo "DOCKER_OPTS=\"-H tcp://127.0.0.1:2375 -H $DOCKER_SOCK -s overlay2\"" | sudo tee /etc/default/docker > /dev/null
+sudo service docker restart
+sleep 5;
+
+if [ "$EMU" = "on" ]; then
```

Comment:
Do we need this? `EMU` probably stands for "emulation", but why should we emulate anything?

Reply:
When CircleCi didn’t have arm support I would be building in qemu docker for armhf and arm64. 
currently not used anymore